easybuild/easyconfigs/g/Gdk-Pixbuf/Gdk-Pixbuf-2.42.10-GCCcore-12.3.0.eb

```
@@ -0,0 +1,46 @@
+easyblock = 'MesonNinja'
+
+name = 'Gdk-Pixbuf'
+version = '2.42.10'
+
+homepage = 'https://docs.gtk.org/gdk-pixbuf/'
+description = """
+ The Gdk Pixbuf is a toolkit for image loading and pixel buffer manipulation.
+ It is used by GTK+ 2 and GTK+ 3 to load and manipulate images. In the past it
+ was distributed as part of GTK+ 2 but it was split off into a separate package
+ in preparation for the change to GTK+ 3.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['ee9b6c75d13ba096907a2e3c6b27b61bcd17f5c7ebeab5a5b439d2f2e39fe44b']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('Meson', '1.1.1'),
+    ('Ninja', '1.11.1'),
+    ('pkgconf', '1.9.5'),
+    ('GObject-Introspection', '1.76.1'),
+]
+
+dependencies = [
+    ('GLib', '2.77.1'),
+    ('libjpeg-turbo', '2.1.5.1'),
+    ('libpng', '1.6.39'),
+    ('LibTIFF', '4.5.1'),
```

We already have an EasyConfig for LibTIFF v4.5.0: https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.5.0-GCCcore-12.3.0.eb

Usually we try to use one version per toolchain generation, unless there is the requirement for a newer versions.

```suggestion
    ('LibTIFF', '4.5.0'),
```

---

Oh, I overlooked that...

Indeed, we're already using LibTIFF 4.5.0 in several easyconfigs using `GCCcore/12.3.0`, so we should stick to that...

---

I changed the dependency for this package accordingly and will retract my PR for libTIFF 4.5.1
https://github.com/easybuilders/easybuild-easyconfigs/pull/18627